gio/src/subclass/async_initable.rs

```diff
@@ -100,12 +102,12 @@ unsafe extern "C" fn async_initable_init_async<T: AsyncInitableImpl>(
 ) {
     let instance = &*(initable as *mut T::Instance);
     let imp = instance.imp();
-    let cancellable = from_glib_borrow::<_, Option<Cancellable>>(cancellable);
+    let cancellable = Option::<Cancellable>::from_glib_none(cancellable);
```

Comment:
Why?

Reply:
Because I need to pass it by value to the cancellable future